cmd/minishift/cmd/start.go

```diff
@@ -221,16 +223,29 @@ func handleProxies() *util.ProxyConfig {
 	return proxyConfig
 }
 
+func getSlice(key string) []string {
+	if viper.IsSet(key) {
+		return viper.GetStringSlice(key)
+	}
+	return nil
+}
+
+func determineInsecureRegistry(key string) []string {
+	s := getSlice(key)
+	return append(s, defaultInsecureRegistry...)
```

Comment:
@hferentschik yes but that is already handed by docker daemon, I tested this scenario explicit and worked as expected.